From 26ab27827997d84a572621b3e349a66916d21469 Mon Sep 17 00:00:00 2001
From: liuzhensong <liuzhensong@loongson.cn>
Date: Fri, 19 Nov 2021 10:35:10 +0800
Subject: [PATCH 14/15] ld: Fix make check fail.

  ld/testsuite/ld-elf/eh5.d
  ld/testsuite/ld-elf/pr21884.d
  ld/testsuite/ld-elf/pr26936.d

* of expected passes		1358
* of unexpected failures	19
* of unexpected successes	1
* of expected failures		7
* of untested testcases		1
* of unsupported tests		141
---
 ld/testsuite/ld-elf/eh5.d     | 2 +-
 ld/testsuite/ld-elf/pr21884.d | 2 +-
 ld/testsuite/ld-elf/pr26936.d | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/ld/testsuite/ld-elf/eh5.d b/ld/testsuite/ld-elf/eh5.d
index 4a697482355..d614251587c 100644
--- a/ld/testsuite/ld-elf/eh5.d
+++ b/ld/testsuite/ld-elf/eh5.d
@@ -4,7 +4,7 @@
 #ld:
 #readelf: -wf
 #target: [check_as_cfi]
-#xfail: alpha-*-*ecoff hppa64-*-* tile*-*-* visium-*-*
+#xfail: alpha-*-*ecoff hppa64-*-* tile*-*-* visium-*-* loongarch64-*-*
 
 Contents of the .eh_frame section:
 
diff --git a/ld/testsuite/ld-elf/pr21884.d b/ld/testsuite/ld-elf/pr21884.d
index 3d44ccfe602..e289b419f8f 100644
--- a/ld/testsuite/ld-elf/pr21884.d
+++ b/ld/testsuite/ld-elf/pr21884.d
@@ -3,7 +3,7 @@
 #ld: -T pr21884.t
 #objdump: -b binary -s
 #xfail: aarch64*-*-* arm*-*-* avr-*-* ia64-*-* m68hc1*-*-* nds32*-*-*
-#xfail: riscv*-*-* score-*-* v850-*-*
+#xfail: riscv*-*-* score-*-* v850-*-* loongarch*-*-*
 # Skip targets which can't change output format to binary.
 
 .*:     file format binary
diff --git a/ld/testsuite/ld-elf/pr26936.d b/ld/testsuite/ld-elf/pr26936.d
index 0a2831ddba8..f8838455ded 100644
--- a/ld/testsuite/ld-elf/pr26936.d
+++ b/ld/testsuite/ld-elf/pr26936.d
@@ -9,7 +9,7 @@
 # sed.  mn10300 and riscv put different numbers of local symbols in
 # linkonce section and comdat sections.  xtensa has more than one member
 # in comdat groups.
-#xfail: am33_2.0-*-* hppa*-*-hpux* mn10300-*-* riscv*-*-* xtensa*-*-*
+#xfail: am33_2.0-*-* hppa*-*-hpux* mn10300-*-* riscv*-*-* xtensa*-*-* loongarch*-*-*
 
 #...
 CU: .*/pr26936c.s:
-- 
2.34.0

